### PR TITLE
refactor(test, Fiji): migrate to python

### PR DESCRIPTION
**Pull request by Rouein**
This script increments unlike the R script(batch), due to the source.
I scraped Historical data but to fill the gaps and `Cumulative total` manual work is needed. I live that part to whichever resigned soul that has time.

### CODE REVIEW

**Review by camappel — APPROVED** [review]
LGTM

File: scripts/scripts/testing/automations/incremental/fiji.py [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+
+    def _get_text_from_url(self, url: str) -> str:
+        """Extract text from the url."""
+        soup = get_soup(url)
+        text = soup.get_text(strip=True).replace("\n", " ").lower()
+        return text
+
+    def _parse_metrics(self, text: str) -> int:
+        """Get metrics from news text."""
+        count = re.search(self.regex["count"], text).group(1)
```
Comment (camappel) [inline scripts/scripts/testing/automations/incremental/fiji.py:86]: Set to Nan if blank

Comment (Rouein) [reply scripts/scripts/testing/automations/incremental/fiji.py:86]: Actually more has to change!
I will push the revised version.

**Review by camappel — APPROVED** [review]
LGTM. Will merge pending @lucasrodes 's approval

**Review by lucasrodes — APPROVED** [review]
LGTM - Thanks!

**Pull request merged**